src/views/AssignPickerModal.vue

```diff
@@ -190,54 +190,33 @@ export default defineComponent({
     },
     async findPickers() {
       this.isLoading = true;
-
       let inputFields = {}
       this.pickers = []
 
       if(this.queryString.length > 0) {
-        inputFields = {
-          firstName_value: this.queryString,
-          firstName_op: 'contains',
-          firstName_ic: 'Y',
-          firstName_grp: '1',
-          externalId_value: this.queryString,
-          externalId_op: 'contains',
-          externalId_ic: 'Y',
-          externalId_grp: '2',
-          lastName_value: this.queryString,
-          lastName_op: 'contains',
-          lastName_ic: 'Y',
-          lastName_grp: '3',
-          partyId_value: this.queryString,
-          partyId_op: 'contains',
-          partyId_ic: 'Y',
-          partyId_grp: '4',
-          groupName_value: this.queryString,
-          groupName_op: 'contains',
-          groupName_ic: 'Y',
-          groupName_grp: '5'
-        }
+        inputFields = `firstName:*${this.queryString}* OR externalId:*${this.queryString}* OR lastName:*${this.queryString}* OR partyId:*${this.queryString}* OR groupName:*${this.queryString}*`
```

Comment:
Rename inputFields to query, also you need not to specify fields here as we will use edismax parser and specify query fields.

Split queryString from space and prepare a string like below: 
let say queryString="Steve Flemming"
query  = (*Steve* OR *Flemming*) OR "Steve Flemming"^100